package/boot/uboot-mvebu/Makefile

```diff
@@ -29,8 +29,16 @@ define U-Boot/clearfog
   UBOOT_IMAGE:=u-boot-spl.kwb
 endef
 
+define U-Boot/uDPU
+  NAME:=Methode uDPU
+  BUILD_DEVICES:=methode_udpu
+  BUILD_SUBTARGET:=cortexa53
+  UBOOT_IMAGE:=u-boot.bin
```

Comment:
This is unnecessary, it's already the default.

Reply:
You're right, I removed UBOOT_IMAGE line.